platform/linux-generic/odp_ipsec.c

```diff
@@ -2250,6 +2250,21 @@ int odp_ipsec_out_inline(const odp_packet_t pkt_in[], int num_in,
 		ipsec_sa = ipsec_out_single(pkt, sa, &pkt, opt, true, &status);
 		ODP_ASSERT(NULL != ipsec_sa);
 
+		if (status.warn.soft_exp_packets || status.warn.soft_exp_bytes) {
+			if (!odp_atomic_load_u32(&ipsec_sa->status_notify)) {
```

Comment:
It would be good to add a comment that the race condition here is intentional, e.g.:

```
	/*
	 * Another thread may have sent the notification by now but we do
	 * not care since sending duplicate expiry notifications is allowed.
	 */
	rc = _odp_ipsec_status_send(ipsec_sa->queue,
```

Reply:
Ack. Updated with v9.